pkg/inference/scheduling/scheduler.go

```diff
@@ -238,8 +337,17 @@ func (s *Scheduler) handleOpenAIInference(w http.ResponseWriter, r *http.Request
 		s.tracker.TrackModel(model)
 	}
 
+	// Compute the model name that will be used to key the runner. For
+	// passthrough backends, we'll just use a dummy name (though we'll still
+	// record the real name below) so that we don't need to allocate multiple
+	// runners (one for each model used by a passthrough backend).
+	runnerModel := request.Model
+	if backendMode == inference.BackendModePassthrough {
+		runnerModel = "passthrough"
```

Comment:
```suggestion
		runnerModel = inference.BackendModePassthrough.String()
```
?

Reply:
I like this idea, but I don't want to couple it to the mode name (even if it's the same for now).  I'll add an additional constant.